hosted/helpers/helper_common.go

```diff
@@ -86,13 +85,12 @@ func CommonBeforeSuite(cloud string) (Context, error) {
 		Expect(err).To(BeNil())
 	}
 
-	clusterCleanup, _ := strconv.ParseBool(os.Getenv("DOWNSTREAM_CLUSTER_CLEANUP"))
```

Comment:
unintended removal ?

Reply:
Nope. `clusterCleanup` is already defined in structs.go.